aten/src/ATen/native/xnnpack/ChannelShuffle.cpp

```diff
@@ -0,0 +1,104 @@
+#ifdef USE_XNNPACK
+
+#include <ATen/native/xnnpack/Common.h>
+#include <ATen/native/utils/Factory.h>
+
+namespace at {
+namespace native {
+namespace xnnpack {
+
+bool use_channel_shuffle(
+    const Tensor& input,
+    const int64_t groups) {
+  using namespace internal;
+
+  // Here are the list of conditions required for this code path to be taken:
+  // * Input must be 4D CPU float tensor with no gradients.
+  // * output_max must be greater than output_min.
```

Comment:
What is the output_max comment? I dont see it being used anywhere here.

Reply:
Sorry, I need to remove that.